KinConnections/resources/routes.py

```diff
@@ -1,10 +1,12 @@
-from .connector import Connector, Connectors
 from .auth import Login, Logout, Signup, SignupConnectee
+from .connector import Connector, ConnectorExternal, Connectors
+
 
 def initialize_routes(api):
  api.add_resource(Login, '/login')
  api.add_resource(Logout, '/logout')
  api.add_resource(Signup, '/signup')
  api.add_resource(SignupConnectee, '/signup/connectee')
  api.add_resource(Connectors, '/connectors')
- api.add_resource(Connector, '/connector/<id>')
\ No newline at end of file
+ api.add_resource(Connector, '/connector/<id>')
+ api.add_resource(ConnectorExternal, '/external/connector')
```

Comment:
nit: similar as before. since this is an actual API can we change the path to be `api/external/connector` to separate it from other routes.